pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -107,10 +108,19 @@ const annBindCompleted = "pv.kubernetes.io/bind-completed"
 // pre-bound). Value of this annotation does not matter.
 const annBoundByController = "pv.kubernetes.io/bound-by-controller"
 
-// annClass annotation represents a new field which instructs dynamic
-// provisioning to choose a particular storage class (aka profile).
-// Value of this annotation should be empty.
-const annClass = "volume.alpha.kubernetes.io/storage-class"
```

Comment:
Actually I reviewed docs/devel/api_changes.md and I think we *could* just abandon alpha, but it would be nice if we didn't.  How awful would it be to support both semantics for 1 release?

@bgrant0607